docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,290 @@
+        ),
+        "w",
+        encoding="utf-8",
+    ) as f:
+        f.write(doc_text)
+
+    if i % int(len(partial_docs) * 0.05) == 0:
+        progress = (i + 1) / len(partial_docs)
+        print(f"Progress: {progress:.2%}")
+```
+
+## Create the Manifest File
+
+The **manifest file** keeps track of document metadata for the dataset. We need to fill some fields so that paperqa doesn’t try to get metadata using llm calls. This will make the indexing process faster.
+
+```python
+manifest = partial_docs.copy()
+manifest["file_location"] = manifest["doc_id"].apply(lambda x: f"files/{x}.txt")
+manifest["doi"] = ""
+manifest["title"] = manifest["doc_id"]
+manifest["key"] = manifest["doc_id"]
+manifest["docname"] = manifest["doc_id"]
+manifest["citation"] = "_"
+manifest.drop(columns=["doc_id", "doc_text"], inplace=True)
+manifest.to_csv(
+    os.path.join(lfrqa_directory, "science_docs_for_paperqa", "manifest.csv"),
+    index=False,
+)
+```
+
+## Filter and Save Questions
+
+Finally, we load the questions and filter them to ensure we only include questions that reference the selected documents:
+
+```python
+questions_df = pd.read_json(
+    os.path.join(rag_qa_benchmarking_dir, "annotations_science_with_citation.jsonl"),
+    lines=True,
+)
+partial_questions = questions_df[
+    questions_df.gold_doc_ids.apply(
+        lambda ids: all(id < amount_of_docs_to_use for id in ids)
+    )
+]
+partial_questions.to_csv(
+    os.path.join(lfrqa_directory, "questions.csv"),
+    index=False,
+)
+```
+
+## Install paperqa
+
+From now on, we will be using the paperqa library, so we need to install it:
+
+```bash
+pip install paper-qa
+```
+
+## Index the documents
+
+Copy the following to a file and run it. Feel free to adjust the concurrency as you like.
+
+PaperQA builds the index every time a question is asked and new files are present. So running this will build the index for you.
+
+You don’t need any llm api keys for building the index, but you do need them to answer questions.
+
+Remember that this process is quick for small portions of the dataset, but can take around 3 hours for the whole dataset.
+
+```python
+import os
+
+from paperqa import Settings, ask
+from paperqa.settings import AgentSettings, IndexSettings, ParsingSettings
+
+settings = Settings(
+    agent=AgentSettings(
+        index=IndexSettings(
+            name="lfrqa_science_index",
+            paper_directory=os.path.join(
+                "data", "rag-qa-benchmarking", "lfrqa", "science_docs_for_paperqa"
+            ),
+            index_directory=os.path.join(
+                "data", "rag-qa-benchmarking", "lfrqa", "science_docs_for_paperqa_index"
+            ),
+            manifest_file="manifest.csv",
+            concurrency=10_000,
+            batch_size=10_000,
+        )
+    ),
+    parsing=ParsingSettings(
+        use_doc_details=False,
+        defer_embedding=True,
+    ),
+)
+answer_response = (
+    ask(
```

Comment:
If this is about indexing, why are we using `ask`?

Can we just use `build_index`?